test/clj/nnts2/note/functional_test.clj

```diff
@@ -12,30 +11,54 @@
 (deftest create-note-success
   (testing "Should create note successfully for valid params"
     (let [request {:nnts-user 1 :body {:title "note-title" :content "note-content"}}
-          {:keys [body status]} (create request)
+          {:keys [body status]} (handler/create request)
           input (into (:body request) {:created-by-id 1})]
       (is (= status 200))
       (is (= input (select-keys body [:title :content :created-by-id]))))))
 
 (deftest create-note-failure
   (testing "Should receive 400 error with string explanation for an integer title"
     (let [request {:nnts-user 1 :body {:title 12344 :content "note-content"}}
-          {:keys [body status]} (create request)]
+          {:keys [body status]} (handler/create request)]
       (is (= status 400))
       (is (string? body))))
   (testing "Should receive 400 error for missing title"
     (let [request {:nnts-user 1 :body {:content "only-content"}}
-          {:keys [status body]} (create request)]
+          {:keys [status body]} (handler/create request)]
       (is (= status 400))
       (is (string? body))))
   (testing "Should receive 400 error for missing content"
     (let [request {:nnts-user 1 :body {:title "only-titlet"}}
-          {:keys [status body]} (create request)]
+          {:keys [status body]} (handler/create request)]
       (is (= status 400))
       (is (string? body))))
   (testing "Should throw postgres exception  without nnts-user"
     (let [request {:body {:title "no user" :content "no user"}}]
-      (is (thrown?  org.postgresql.util.PSQLException (create request)))))
+      (is (thrown?  org.postgresql.util.PSQLException (handler/create request)))))
   (testing "Should throw postgres exception  without existing nnts-user"
     (let [request {:nnts-user 2 :body {:title "non-existent user" :content "non-existent user"}}]
-      (is (thrown?  org.postgresql.util.PSQLException (create request))))))
+      (is (thrown?  org.postgresql.util.PSQLException (handler/create request))))))
+
+(deftest get-notes
+  (testing "creation of one note should give one note on get"
+    (let [request {:nnts-user 1 :body {:title "note-title" :content "note-content"}}
+          resp (handler/create request)
+          get-request (dissoc request :body)
+          {:keys [body status]} (handler/getnotes request)
+          note-body-request (into (:body request) {:created-by-id 1})]
+      (is (= status 200))
+      (is (= (count body) 1))
```

Comment:
Maybe break these tests down by layer and write unit tests? Currently, each test here seems to be testing across multiple layers, as a consequence of which it's testing a combination of results rather than testing functions in isolation.